src/hosted-buttons/index.js

```diff
@@ -34,24 +39,59 @@ export const getHostedButtonsComponent = (): HostedButtonsComponent => {
         selector,
       });
 
-      // $FlowFixMe
-      Buttons({
-        hostedButtonId,
-        style,
-        onInit,
-        onClick,
-        createOrder: buildHostedButtonCreateOrder({
-          enableDPoP,
-          hostedButtonId,
-          merchantId,
-        }),
-        onApprove: buildHostedButtonOnApprove({
-          enableDPoP,
+      if (shouldRenderSDKButtons(fundingSources)) {
+        const { flexDirection } = getFlexDirection({ ...style });
+
+        buildButtonContainer({ flexDirection, selector });
```

Comment:
it took me a second to understand that `buildButtonContainer` was appending the newly created elements to the DOM - maybe this could be named `appendButtonContainer`? Totally optional suggestion I'm not that great at naming things

Reply:
`appendButtonContainer` works for me and will make even more sense if NCP teams is giving us an id to append to